ProjectName/src/navigation/navtabs.styles.ts

```diff
@@ -10,6 +10,12 @@ const navTabsStyle = StyleSheet.create({
     fontSize: 18,
     fontWeight: '500',
   },
+  PaymentStyle: {
```

Comment:
Please, stick to common `js` naming. Only `Types` and `Classes` should be Upperacased.
So here it should be `paymentStyle`